[3.4.0RC1] NPE on FileDetailSharingFragment.setShareWithUserInfo

via GPlay Android 8.x:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2946)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3046)
  at android.app.ActivityThread.-wrap11 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1688)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6798)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:240)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:767)
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.ui.fragment.FileDetailSharingFragment.setShareWithUserInfo (FileDetailSharingFragment.java:239)
  at com.owncloud.android.ui.fragment.FileDetailSharingFragment.setupView (FileDetailSharingFragment.java:196)
  at com.owncloud.android.ui.fragment.FileDetailSharingFragment.onCreateView (FileDetailSharingFragment.java:175)
  at android.support.v4.app.Fragment.performCreateView (Fragment.java:2439)
  at android.support.v4.app.FragmentManagerImpl.moveToState (FragmentManager.java:1460)
  at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState (FragmentManager.java:1784)
  at android.support.v4.app.FragmentManagerImpl.moveToState (FragmentManager.java:1852)
  at android.support.v4.app.FragmentManagerImpl.dispatchStateChange (FragmentManager.java:3269)
  at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated (FragmentManager.java:3229)
  at android.support.v4.app.FragmentController.dispatchActivityCreated (FragmentController.java:201)
  at android.support.v4.app.FragmentActivity.onStart (FragmentActivity.java:620)
  at android.support.v7.app.AppCompatActivity.onStart (AppCompatActivity.java:178)
  at com.owncloud.android.ui.activity.BaseActivity.onStart (BaseActivity.java:188)
  at com.owncloud.android.ui.activity.DrawerActivity.onStart (DrawerActivity.java:1379)
  at com.owncloud.android.ui.activity.FileActivity.onStart (FileActivity.java:189)
  at com.owncloud.android.ui.activity.FileDisplayActivity.onStart (FileDisplayActivity.java:2564)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1412)
  at android.app.Activity.performStart (Activity.java:7020)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2909)
```

--- a/src/main/java/com/owncloud/android/ui/fragment/FileDetailSharingFragment.java
+++ b/src/main/java/com/owncloud/android/ui/fragment/FileDetailSharingFragment.java
@@ -172,6 +172,10 @@ public class FileDetailSharingFragment extends Fragment implements UserListAdapt
         View view = inflater.inflate(R.layout.file_details_sharing_fragment, container, false);
         unbinder = ButterKnife.bind(this, view);
 
+        if (fileDataStorageManager == null) {
+            fileDataStorageManager = new FileDataStorageManager(account, fileDisplayActivity.getContentResolver());
+        }
+
         setupView();
 
         return view;

